core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -889,7 +889,7 @@ public PropServerConfiguration(
             this.lineTcpEnabled = getBoolean(properties, env, PropertyKey.LINE_TCP_ENABLED, true);
             if (lineTcpEnabled) {
                 // obsolete
-                lineTcpNetConnectionLimit = getInt(properties, env, PropertyKey.LINE_TCP_NET_ACTIVE_CONNECTION_LIMIT, 256);
+                lineTcpNetConnectionLimit = getInt(properties, env, PropertyKey.LINE_TCP_NET_ACTIVE_CONNECTION_LIMIT, 64);
```

Comment:
I saw ILP connection limit reached few times recently for some users, it was connected to slow ingestion but with this it will happen even faster.
I think this is better to be a function of CPU cores.

Reply:
Having it as a function of CPU cores wouldn't be very practical. It could be a function of the I/O worker pool (or shared worker pool) size, but then users might be having hard times understanding how this limit is calculated. I'd keep it as a constant number.

As for 256 connections as the default value, this seems to be very high for an average server. If the lower value helps problems to manifest faster, that's not bad.

WDYT?

Reply:
What about those who use QuestDB on the powerful servers at the moment? The new limit may not work for them after update.

Reply:
Makes sense. Changing this default is something to be done in 7.0 release. Reverted default for `line.tcp.net.connection.limit` in a1464e244b0e316cc9921a9c4825c593c8485559 to be on the safer side.